README.md

```diff
@@ -26,7 +26,7 @@ OKD: The Origin Community Distribution of Kubernetes
 * Multi-tenancy support, including team and user isolation of containers, builds, and network communication.
   * Allow developers to run containers securely with fine-grained controls in production.
   * Limit, track, and manage the developers and teams on the platform.
-* Integrated Docker registry, automatic edge load balancing, cluster logging, and integrated metrics.
+* Integrated container registry, automatic edge load balancing, cluster logging, and integrated metrics.
```

Comment:
imho it's an image registry, not a container registry.

Reply:
It might literally be an image registry, but I have never heard a customer call it such.  Perhaps a
container image registry.

Reply:
which is probably our fault for describing it badly(and creating something called the "redhat container catalog").  we should stop reinforcing it.

Reply:
So how do I go forward.  Do you want me to change it to container image registry?

Reply:
i guess so. i was hoping some other people would weigh in.